modules/apps/29-fee/keeper/keeper_test.go

```diff
@@ -98,11 +98,14 @@ func (suite *KeeperTestSuite) TestFeesInEscrow() {
 	packetID := channeltypes.NewPacketId(suite.path.EndpointA.ChannelConfig.PortID, suite.path.EndpointA.ChannelID, 1)
 	fee := types.NewFee(defaultReceiveFee, defaultAckFee, defaultTimeoutFee)
 
+	var packetFees []types.PacketFee
 	for i := 1; i < 6; i++ {
 		packetFee := types.NewPacketFee(fee, suite.chainA.SenderAccount.GetAddress().String(), nil)
-		suite.chainA.GetSimApp().IBCFeeKeeper.EscrowPacketFee(suite.chainA.GetContext(), packetID, packetFee)
+		packetFees = append(packetFees, packetFee)
```

Comment:
Can we do this the same way you have [here](https://github.com/cosmos/ibc-go/pull/1252/files#diff-576b5df34a660fb3ea0475bd98de7e9a9e14617a8d67426801fe84f32b1ce2d8R394)?

i.e.

```
packetFee := types.NewPacketFee(fee, suite.chainA.SenderAccount.GetAddress().String(), nil)
packetFees := []types.PacketFee{packetFee, packetFee, packetFee, packetFee, packetFee}
suite.chainA.GetSimApp().IBCFeeKeeper.SetFeesInEscrow(suite.chainA.GetContext(), packetID, types.NewPacketFees(packetFees))

```

Reply:
Updated!